avocado/core/suite.py

```diff
@@ -41,6 +40,49 @@ class TestSuiteStatus(Enum):
     UNKNOWN = object()
 
 
+def resolutions_to_runnables(resolutions, config):
```

Comment:
Hi @clebergnu can you explain to me why you moved this method from `core.utils.py`? IMO this a utility which should be in `core.utils.py`.

Reply:
If I recall correctly, this was because of old code when the nrunner was not integrated into the jobs API.

Personally, I prefer to avoid lots of `utils` libraries spread all over the code. If this function is used just inside the `suite` now, let's keep it here or move it inside some class that makes sense.